bricks/forms/src/upload-files-v2/UploadFilesV2.tsx

```
@@ -130,11 +134,27 @@ export function RealUploadFile(
+          i18n.t(`${NS_FORMS}:${K.NO_SUPPORT_FILE_TYPE}`)
+        );
         return new Promise((_resolve, reject) => {
           reject(new Error(i18n.t(`${NS_FORMS}:${K.NO_SUPPORT_FILE_TYPE}`)));
         });
       }
     }
+    // 正则校验文件名
+    if (props.fileNamePattern && props.fileNamePattern?.test(file?.name)) {
+      props.onCustomError?.(
+        "name",
+        i18n.t(`${NS_FORMS}:${K.FILE_NAME_VALIDATE_MESSAGE_LOG}`)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)